pendulum/pendulum.py

```diff
@@ -70,16 +70,15 @@ def _safe_create_datetime_zone(cls, obj):
         elif isinstance(obj, datetime.tzinfo) and not isinstance(obj, Timezone):
             # pytz
             if hasattr(obj, 'localize'):
-                obj = obj.zone
-            else:
-                # We have no sure way to figure out
-                # the timezone name, we raise an error
+                return cls._timezone(obj.zone)
 
-                raise ValueError('Unsupported timezone {}'.format(obj))
+            return FixedTimezone(obj.utcoffset(None).total_seconds())
 
-        tz = cls._timezone(obj)
-
-        return tz
+        try:
```

Comment:
We don't need the `try/except` statement here since a `TimezoneNotFound` error will be raised when trying to load an invalid timezone.